test/app/view/web/WebChessBoardView.spec.ts

```diff
@@ -133,3 +133,61 @@ describe('Web Chess Board View with starting pieces positions', () => {
     expect(a5Square).not.toHaveClass('square--possibleMove');
   });
 });
+
+describe('Web Chess Board View during move and captures methods', () => {
+  const publishViewEventMock = jest.fn();
+  const viewEventBus: ViewEventBus = {
+    listenOn: jest.fn(),
+    publish: publishViewEventMock,
+  };
+  const chessBoardView: ChessBoardView = new WebChessView(viewEventBus);
+
+  it('move pawn from square A2 to A3', async () => {
+    const piecesPositions: PiecesBoardPositions = {
+      A2: { name: 'Pawn', side: Side.WHITE },
+    };
+    chessBoardView.showChessBoard(piecesPositions);
+    const a2Square = await screen.findByTestId('a2');
```

Comment:
@nowakprojects  nie rozumiem trochę, czemu mam niby nagle dwa pola o id='a2'? To jakoś tworzy mi kopię, czy o co chodzi?
![image](https://user-images.githubusercontent.com/31566345/107853230-08722900-6e15-11eb-8af3-4ef974be8e3f.png)